.eslintrc.js

```diff
@@ -9,7 +9,9 @@ module.exports = {
       rules: {
         'react-perf/jsx-no-new-array-as-prop': 'off',
         'react-perf/jsx-no-new-function-as-prop': 'off',
-        'react-perf/jsx-no-new-object-as-prop': 'off'
+        'react-perf/jsx-no-new-object-as-prop': 'off',
+        'space-infix-ops': 'off',
```

Comment:
can we add this to https://github.com/yext/slapshot-tools/tree/main/packages/eslint-config-slapshot and publish a new minor?

Reply:
Okay, I just made a PR in the slapshot-tools repo where I added the rule